libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -523,17 +523,20 @@ private static boolean resizeImageAndWriteToStream(Context context,
 
         boolean result = bmpRotated.compress(fmt, quality, outStream);
         bmpRotated.recycle();
+        bmpResized.recycle();
```

Comment:
I think we can safely leave it up to the Android system. May I suggest we bring someone else's opinion on this particular?